### PR TITLE
[Equalizer] Bright analyzer colors, opacity increased

### PR DESCRIPTION
**Pull request by GingkathFox**
I dont really like how dark and transparent the analyzer colors are. For me, it's hard to actually see the difference between the input and the output. This pull changes the in color and makes both colors a bit more opaque.

### CODE REVIEW

File: plugins/Eq/EqControlsDialog.cpp [hunk 1]
```diff
@@ -55,10 +55,10 @@ EqControlsDialog::EqControlsDialog( EqControls *controls ) :
 
 	EqSpectrumView * inSpec = new EqSpectrumView( &controls->m_inFftBands, this );
 	inSpec->move( 26, 17 );
-	inSpec->setColor( QColor( 54, 45, 142, 150 ) );
+	inSpec->setColor( QColor( 63, 202, 141, 245 );
```
Comment (PhysSong) [inline plugins/Eq/EqControlsDialog.cpp]: A `(` is missing...

Comment (GingkathFox) [reply plugins/Eq/EqControlsDialog.cpp]: oof, lemmie edit

Comment (GingkathFox) [reply plugins/Eq/EqControlsDialog.cpp]: fixed, how the heck did i do that 🤔

Comment (GingkathFox) [reply plugins/Eq/EqControlsDialog.cpp]: wait a sec
why is it not displaying

Comment (GingkathFox) [reply plugins/Eq/EqControlsDialog.cpp]: o wait, its under xD
i need to figure out this pull request thing

**Comment by GingkathFox** [comment]
[Uploading EqControlsDialog.cpp…]()

**Comment by tresf** [comment]
With any UI-only changes, a before and after helps.

Before: screen shot 2019-01-14 at 10 34 33 pm
After: screen shot 2019-01-14 at 10 38 10 pm

**Comment by Spekular** [comment]
If you want to make the input and output more distinct, it's an odd choice
to make them both blue.

I tried to get reference on what colors other EQs use. A quick search shows
that Pro-Q, Logic X's EQ, and izotope's EQs (inside neutron/ozone) use a
colorless waveform, instead using color to show how individual bands are
affecting the eq curve. Ableton's EQ forgoes color coding entirely and FL
studio's EQ uses an entirely different visualization.

**Comment by tresf** [comment]
Let's please focus on accepting or denying this PR.  If the OP's concerns are valid, focuson on them.  Enhancements described above are out of scope of our stable branch.

**Comment by Spekular** [comment]
I'm not describing any enhancements, though.


1. I have no trouble seeing the two waveforms in the current version, nor
in this edited version. However, in the edited version the waveforms are
less distinct. Perhaps calling it an "odd choice" was too diplomatic? In
the new version, it's harder to tell which waveform is in and which is out.


2. If the color is to be changed, I thought it would be helpful to see how
it's done elsewhere rather than figuring things out from scratch. This
presents two options:


A: We distinguish the signals in some way other than color, as seems to be
the trend. This would mean this PR should be closed rather than changing a
set of colors that will be scrapped.


B: We distinguish the signals by color. This means point 1 is valid and the
PR may need to be edited before being accepted.


Which part of this isn't relevant to the PR?

**Comment by tresf** [comment]
> Enhancements described above are out of scope of our stable branch.

> > Which part of this isn't relevant to the PR?

Out of scope does not negate relevance, but establishing scope forces decision.

* Do you agree the current colors are too transparent
* Do you agree the current colors are hard to differentiate

The PR touches **two** lines of code and makes no mention of making things colorless, making the bands affect color, etc.  The points are extremely relevant, but they're out of scope, mostly because they lead to "endless" discussion.

Let's focus on @GingkathFox's PR and how he can make it better please.

**Comment by RoxasKH** [comment]
In my personal opinion, i really like the actual visualizer colors. I think the new ones are a bit more confusionary, also they don't fit the UI so much. So i'd maintain the actual ones, but it's not only my decision.

**Comment by GingkathFox** [comment]
@Spekular  Oh crap, i think i accidentally made both blue. The inputs supposed to be a green color, lemmie change that.
@tresf cmake and make on MacOS hate me, absolutely nothing will build without a fatal error of some sort. Ill see if i can build on Ubuntu. 
@RoxasKH see above xp

**Comment by GingkathFox** [comment]
Lemmie try to build on Ubuntu, ill get back to yall in a bit.

**Comment by GingkathFox** [comment]
Wasnt able to build, ill try something tomorrow.

**Comment by GingkathFox** [comment]
I cant get make to install on ubuntu 18.04, sorry. Can someone check the color change for me?

**Comment by GingkathFox** [comment]
In:
In
Out:
Out
Both:
Both

**Comment by GingkathFox** [comment]
@RebeccaDeField i dunno, i didnt really like the blue&green combo. ill try it.

**Comment by GingkathFox** [comment]
screen shot 2019-01-31 at 9 01 19 pm

**Comment by RebeccaDeField** [comment]
@GingkathFox I didn't mean to use the old colors, just the same hues. Blue and green should still be alright as long as it has enough contrast (one is lighter than the other) and is brighter to fulfill the purpose of this pr 👍

**Comment by RebeccaDeField** [comment]
@GingkathFox would be a little more complicated to change the hue using RGB. In inkscape, try switching to HSL in fill and stroke.
dialog_fillstroke_hsl

Then only edit the Lightness value, leave the hue/saturation and you should be good to go.

**Comment by GingkathFox** [comment]
im running MacOS >w<
ive been using[this](http://colorizer.org), maybe i can just change the hue of the HSL value and copy the RGB?

**Comment by tresf** [comment]
> In inkscape, try switching to HSL in fill and stroke.

> > im running MacOS >w<

Translation: macOS can't run Inkscape (they claim it works with X11 server, but it doesn't).  @GingkathFox you have a Linux VM, muscle up. 💪 😄

**Comment by GingkathFox** [comment]
@tresf i dont wanna reboot pc to switch thoooo XDDD

**Comment by RebeccaDeField** [comment]
> ive been using this, maybe i can just change the hue of the HSL value and copy the RGB?

@GingkathFox Any color program using HSL will work fine, only assumed inkscape because that's where the source files for the colors are contained but the program choice is arbitrary as long as you can grab the colors you need.

**Comment by GingkathFox** [comment]
In:
in
Out: out
In&Out:
in out

**Comment by RebeccaDeField** [comment]
@GingkathFox This looks good, but I am thinking that the In color should be brighter given the purpose fo the PR?

**Comment by GingkathFox** [comment]
XD now i gotta mess with the in color again! Time to fire up Colorizer and ill push the changes later today @RebeccaDeField

**Comment by RebeccaDeField** [comment]
Given that this matches our existing colors and it looks more visible than the previous version of the equalizer, I think we've accomplished our goal here and this has my approval from the design dept.

**Comment by RebeccaDeField** [comment]
Some checks not successful. Anything we can do to fix that or is this ok to merge? Hoping to get this cleared out of the pr requests as it is a pretty minor change.

**Comment by PhysSong** [comment]
> Some checks not successful. Anything we can do to fix that or is this ok to merge?

The failure looks unrelated to changes in this PR. I restarted the Travis-CI build and it should pass in 30 minutes.

**Pull request merged**